src/components/home-page/hero-section/hero-section.vue

```diff
@@ -0,0 +1,92 @@
+<script>
+    import { defineComponent } from 'vue';
+
+    import BaseButton from '@/components/global/base-button.vue';
+
+    export default defineComponent({
+        name: 'HeroSection',
```

Comment:
Thanks a lot!
I'm learning to work with PR's - which is a bit of a pain because my git workflow until now may have been a little chaotic... :) - but I'll do my best to keep them short & concise.

Regarding the indentations & file naming, if it is a convention used at Monte (among others) then the sooner I adapt it the better, thank You for pointing it out.
I'll change all files after rebase so I don't run into any conflicts.